internal/controllers/rpc.go

```
@@ -0,0 +1,65 @@
+package controllers
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/NodeFactoryIo/vedran/internal/rpc"
+	log "github.com/sirupsen/logrus"
+)
+
+func (c ApiController) RPCHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	defer r.Body.Close()
+	reqBody, _ := ioutil.ReadAll(r.Body)
```

Are you sure that you can ignore this error ?